libtransmission/transmission.h

```diff
@@ -35,6 +35,7 @@ using tr_tracker_id_t = uint32_t;
 using tr_torrent_id_t = int;
 using tr_bytes_per_second_t = size_t;
 using tr_kilobytes_per_second_t = size_t;
+using tr_mode_t = uint32_t;
```

Comment:
mode_t is defined as `uint16_t` on DARWIN.
I think `uint16_t` is enough.

Reply:
Sounds good. Fixed in cdf45da762c5f4c6c23e1a8e90afa2c8a38cbbc6